metal/v1/test/api_vrfs_test.go

```diff
@@ -0,0 +1,95 @@
+/*
+Metal API
+
+Testing VRFsApiService
+
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech);
+
+package v1
+
+import (
+	"context"
+	"testing"
+
+	openapiclient "github.com/equinix-labs/metal-go/metal/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_v1_VRFsApiService(t *testing.T) {
+	configuration := openapiclient.NewConfiguration()
+	apiClient := openapiclient.NewAPIClient(configuration)
+
+	t.Run("Test VRFsApiService CreateVrf", func(t *testing.T) {
+		t.Skip("skip test") // remove to run test
```

Comment:
The `make test` task passes now, but every generated test starts with a call to `t.Skip`, so the generated code isn't being exercised.  I don't see a way to tell the generator to remove the `t.Skip` calls, so this may require another post-patch to remove these lines from every test file.

Reply:
Perhaps a `sed _t.Skip("skip test")_d` like the existing `find ${PACKAGE_PREFIX}/${PACKAGE_MAJOR}/test -type f -exec`